modules/installation-osp-creating-network-resources.adoc

```diff
@@ -78,7 +78,7 @@ $ ansible-playbook -i inventory.yaml security-groups.yaml
 +
 [source,terminal]
 ----
-$ ansible-playbook -i inventory.yaml network.yaml
+$ ansible-playbook -i inventory.yaml update-network-resources.yaml
```

Comment:
@MaysaMacedo is this the line you are referring to when you said "we should be updating the network resources like it's done [here](https://github.com/openshift/installer/commit/c670cd90ba116db9120fbb9b857f9f3cc611baae#diff-6048f620b6429cf39395b4c267287d0d9b4ac9da05a2fa17cebc5f2381bba5c3R813)."? If so, what changes need to be made here? It looks as though this is the correct ansible-playbook command per the readme. Thank you